apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs

```diff
@@ -122,4 +122,248 @@ private Operation PollForCompletion(Operation operation, string alias)
+                Assert.Equal(true, instance.ShieldedInstanceConfig.EnableSecureBoot);
+            }
+
+        }
+        [Fact (Skip="resize fails")]
+        public void TestQueryParams(){
+            string projectId = _fixture.ProjectId;
+            string zone = _fixture.Zone;
+            InstanceGroupManagersClient instanceGroupManagersClient = InstanceGroupManagersClient.Create();
+            InstanceTemplatesClient instanceTemplatesClient = InstanceTemplatesClient.Create();
+            var instanceGroupManagerName = _fixture.GenerateResourceId();
+            var instanceTemplateName = _fixture.GenerateResourceId();
+            AttachedDiskInitializeParams init_params = new AttachedDiskInitializeParams
+                {
+                    SourceImage = "projects/debian-cloud/global/images/family/debian-10"
+                };
+            InstanceProperties props = new InstanceProperties{
+                Disks = {
+                            new AttachedDisk{
+                                AutoDelete = true,
+                                Boot = true,
+                                Type = AttachedDisk.Types.Type.Persistent,
+                                InitializeParams = init_params
+                            }
+                        },
+                NetworkInterfaces = {
+                    new NetworkInterface{
+                        Name="default"
+                        }
+                },
+                MachineType = "n1-standard-1"
+            };
+            InstanceTemplate instanceTemplate = new InstanceTemplate
+            {
+                Name = instanceTemplateName,
+                Properties = props
+            };
+            try{
+                Operation insertTemplateOp = instanceTemplatesClient.Insert(
+                    projectId,
+                    instanceTemplate
+                );
+                PollForCompletionGlobalOp(insertTemplateOp, "insert template");
+
+                InstanceGroupManager instanceGroupManager = new InstanceGroupManager{
+                    InstanceTemplate = insertTemplateOp.TargetLink,
+                    TargetSize = 1,
+                    Name = instanceGroupManagerName,
+                    BaseInstanceName = "dotnetgapic"
+                };
+
+                Operation insertIgmOp = instanceGroupManagersClient.Insert(
+                    projectId,
+                    zone,
+                    instanceGroupManager
+                );
+                PollForCompletion(insertIgmOp, "insert group manager");
+
+                Operation resizeOp = instanceGroupManagersClient.Resize(
+                    projectId,
+                    zone,
+                    instanceGroupManagerName,
+                    0
+                );
+
+                PollForCompletion(resizeOp, "resize");
+
+                InstanceGroupManager instanceGroupManagerResized = instanceGroupManagersClient.Get(
```

Comment:
I'd probably use `var` and a shorter variable name (maybe just `fetched`?) and put the whole method call on one line.